frontend/src/routes/(root)/(logged)/workspace_settings/+page.svelte

```diff
@@ -976,27 +988,35 @@
 					</div>
 				</div>
 			</div>
+			<ToggleButtonGroup bind:selected>
+				<ToggleButton value="openai" label="OpenAi" />
+				<ToggleButton value="anthropic" label="Anthropic" />
+				<ToggleButton value="mistral" label="Mistral" />
+			</ToggleButtonGroup>
```

Comment:
I thought the same about UX, however I let it that way so user can see which key is set. For now I added your suggestion so it does actually reset aiResourceInitialPath each time a new option is selected, tell me your though about it.

Reply:
It shouldn't be done on start. There is a "selected" event on <ToggleButtonGroup> and you can reset aiResourceInitialPath when dispatched: on:selected={() => {aiResourceInitialPath = ''}
So it would only be reset when changing the kind of resource. What do you think?

Reply:
You're totally right, it will provide better readability about what actually happens when one event is selected, I have made the fix.